assets/js/lib/test-utils/factories/sapSystems.js

```diff
@@ -18,31 +20,36 @@ const roles = () =>
 export const sapSystemApplicationInstanceFactory = Factory.define(() => ({
   features: roles().join('|'),
   health: healthEnum(),
-  hostId: faker.datatype.uuid(),
-  httpPort: faker.internet.port(),
-  httpsPort: faker.internet.port(),
-  instanceHostname: faker.hacker.noun(),
-  instanceNumber: faker.datatype.number({ min: 10, max: 99 }).toString(),
+  host_id: faker.datatype.uuid(),
```

Comment:
I have changed this factory, as it was before, was not creating data as we store in the `redux` store. It was using as it comes the `GenericSystemDetails` view page.
Just to make it more generic, now it replicates the data coming from the web.